website/docs/observability/index.md

```diff
@@ -20,3 +20,7 @@ Watch a video walk-through of the observability module with one of the module ma
 :::info
 To dive deeper into AWS Observability features take a look at the [One Observability Workshop](https://catalog.workshops.aws/observability/en-US)
 :::
+
+:::info
+Explore a set of opinionated Infrastructure as Code (IaC) modules to help you set up observability for your AWS environments in our [AWS Observability Accelerator for CDK](https://aws-observability.github.io/cdk-aws-observability-accelerator/) and [AWS Observability Accelerator for Terraform](https://aws-observability.github.io/terraform-aws-observability-accelerator/). These modules work with AWS Native services like Amazon CloudWatch and AWS-managed observability services such as Amazon Managed Service for Prometheus, Amazon Managed Grafana and AWS Distro for OpenTelemetry (ADOT).
```

Comment:
should we add the diagram of - Single EKS Cluster Open Source Observability Accelerator, or share some example patterns like - cost monitoring, alerting rules etc, to get people interested and click on the links to learn more ?

Reply:
I agree @jsgoudar, adding links to a few of the patterns.